x/tokenfactory/types/codec.go

```diff
@@ -28,52 +33,27 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&MsgUpdateMasterMinter{},
-	)
-	registry.RegisterImplementations((*sdk.Msg)(nil),
```

Comment:
Yes but `RegisterImplementations` is a variadic function so we can pass all the msg types at once instead of calling the function several times for each msg type.